server/src/routes/api/v1/buildsRouter.js

```diff
@@ -37,10 +37,24 @@ buildsRouter.get("/:id", async (req, res) => {
     try{
         const selectedBuild = await Build.query().findById(id)
         const serializedBuild = await BuildSerializer.getBuildDetails(selectedBuild)
-        res.status(200).json({ build: serializedBuild })
+        return res.status(200).json({ build: serializedBuild })
     } catch(error) {
-        res.status(500).json({errors: error})
+        return res.status(500).json({errors: error})
     }
 })
 
+buildsRouter.delete("/:id", async (req,res) => {
+    const id = req.params.id
+    try {
+        const buildToDelete = await Build.query().findById(id)
```

Comment:
1. we need a check at this endpoint to ensure the currently signed in user, req.user, is the same user who created the build
2. take a look at the objection docs for how to delete by id to reduce the number of queries to the database
https://vincit.github.io/objection.js/guide/query-examples.html#delete-queries
3. take a look at this past question queue question about adding a new migration that includes a cascade effect when deleting a build so that it's associated reviews are deleted without us having to explicitly delete them in this endpoint
https://learn.launchacademy.com/questions/14055